src/component/NodeEditor/NodeEditor.tsx

```diff
@@ -48,17 +50,16 @@ interface Sides {
 export interface FormProps {
     showAdvanced: boolean;
     node: Node;
+    operand: string;
     translating: boolean;
     language: Language;
-    action?: AnyAction;
-    localizations?: LocalizedObject[];
     definition: FlowDefinition;
     config: Type;
     ComponentMap: ComponentMap;
     updateAction(action: AnyAction): void;
     onBindWidget(ref: any): void;
     onBindAdvancedWidget(ref: any): void;
-    updateRouter(node: Node, type: string, previousAction: AnyAction): void;
+    updateRouter: Function;
```

Comment:
Can you explain this change?

Reply:
yes using the generic ‘Function’ type since the value of the prop will one of ‘updateSwitchRouter’, ‘updateSubflowRouter’, ‘updateGroupRouter’ or ‘updateWebhoomRouter’